plugins/modules/cloud/smartos/vmadm.py

```diff
@@ -601,23 +597,17 @@ def vm_state_transition(module, uuid, vm_state):
 
 
 def is_valid_uuid(uuid):
-    if re.match('^[0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12}$', uuid, re.IGNORECASE):
-        return True
-    else:
-        return False
+    return bool(re.match('^[0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12}$', uuid, re.IGNORECASE))
```

Comment:
Here's a very similar one:
```suggestion
    return re.match('^[0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12}$', uuid, re.IGNORECASE) is not None
```